src/openrct2-ui/windows/Guest.cpp

```diff
@@ -1475,19 +1475,19 @@ void window_guest_stats_paint(rct_window* w, rct_drawpixelinfo* dpi)
     y += LIST_ROW_HEIGHT;
     gfx_draw_string_left(dpi, STR_GUEST_STAT_NAUSEA_LABEL, gCommonFormatArgs, COLOUR_BLACK, x, y);
 
-    int32_t nausea = peep->Nausea - 32;
+    int32_t Nausea = peep->Nausea - 32;
```

Comment:
... have you been using find and replace again?

Reply:
wow